js/deer-render.js

```diff
@@ -204,6 +204,255 @@ DEER.TEMPLATES.shadow = (obj, options = {}) => {
     }
 }
 
+DEER.TEMPLATES.statusComment = (obj, options = {}) => {
```

Comment:
These can go in deer-config more cleanly, but this is fine.

Reply:
I see what you're saying.  Some of those templates I added don't use UTILS.  Got ya, yes those would do just fine in deer-config.